app/jobs/onceoff/rename_canned_replies.rb

```diff
@@ -0,0 +1,14 @@
+module Jobs
+  class RenameCannedReplies < Jobs::Onceoff
+    OLD_PLUGIN_NAME = "canned_replies"
+    NEW_PLUGIN_NAME = "discourse-canned-replies"
+
+    def execute_onceoff(args)
+      old_rows = PluginStoreRow.where(plugin_name: OLD_PLUGIN_NAME)
```

Comment:
No need to check the # of rows. Less code is better 😉 

```suggestion
PluginStoreRow.where(plugin_name: NEW_PLUGIN_NAME).delete_all
PluginStoreRow.where(plugin_name: OLD_PLUGIN_NAME).update_all(plugin_name: NEW_PLUGIN_NAME)
```

Reply:
I know, I just wanted to be on the safe side here.

If you think it's better if we don't check, I do not mind. 👍